src/Functions/README.md

```diff
@@ -69,7 +69,7 @@ directive:
 ```
 
 ``` yaml
-branch: main
+commit: 78eac0bd58633028293cb1ec1709baa200bed9e2
```

Comment:
Could you write in description about how to handle the case when the original branch is main branch?